t/16-utils.t

```diff
@@ -118,6 +118,9 @@ is href_to_bugref('https://progress.opensuse.org/issues/1234'), 'poo#1234';
 is bugref_to_href('boo#9876'), '<a href="https://bugzilla.opensuse.org/show_bug.cgi?id=9876">boo#9876</a>';
 is href_to_bugref('https://github.com/foo/bar/issues/1234'), 'gh#foo/bar#1234';
 is href_to_bugref('https://github.com/foo/bar_bar/issues/1234'), 'gh#foo/bar_bar#1234';
+is href_to_bugref('https://github.com/foo6/bar666/issues/1234'), 'gh#foo6/bar666#1234';
+# invalid URL but it is still generated
```

Comment:
The URL on the next line is valid. (It leads to a 404 page but the URL itself is valid.) So just remove that comment.

```suggestion
```